packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -867,11 +867,13 @@ export const prepareVaultManagerKit = (
           const { state } = this;
           const { collateralBrand, totalDebt } = state;
 
-          checkDebtLimit(
-            factoryPowers.getGovernedParams(collateralBrand).getDebtLimit(),
-            totalDebt,
-            toMint,
-          );
+          if (!AmountMath.isEmpty(toMint)) {
```

Comment:
I think this is the job of `checkDebtLimit` itself. It's doc says,
```
 * @param {Amount<'nat'>} toMint
 * @throws if minting would exceed total debt
```
If toMint is empty then it shouldn't throw. 

So instead of this change I suggest updating that helper function and adding whatever docs would have helped to make that the clear solution. This will also require a change to `contractSupport.test.js`. That will unit test this logic, though it's certainly helpful to also have the integration test you make in `vault-collaterlization-test.js`.